.github/workflows/codecov.yml

```diff
@@ -0,0 +1,26 @@
+name: Codecov
+on:
+  pull_request:
+  push:
+
+jobs:
+  codecov:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.19.x]
+        os: [ubuntu-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go-version }}
+          cache: true
+      - name: Tests and coverage (No Tinygo)
+        run: go test -coverprofile=coverage-normal.txt -covermode=atomic ./...     
+      - uses: codecov/codecov-action@v2
```

Comment:
Since we already run tests with coverage / race detector in regression, how about just adding the action after that?

https://github.com/corazawaf/coraza/blob/v3/dev/.github/workflows/regression.yml#L32